CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- `BaseArtifact.to_bytes()` method to convert an Artifact to bytes.
```

Comment:
not: should this be `Artifact value to bytes`?

Reply:
Updated